corehq/apps/analytics/tasks.py

```diff
@@ -148,6 +148,13 @@ def _send_form_to_hubspot(form_id, webuser, cookies, meta):
 
 
 @task(queue='background_queue', acks_late=True, ignore_result=True)
+def update_hubspot_properties(webuser, properties):
+    vid = _get_user_hubspot_id(webuser)
+    if vid:
+        _track_on_hubspot(webuser, properties)
```

Comment:
It looks like it was specified in the original email exchange I had with Jon about this that new users should not be created when the "built_app" event was triggered, only when the "created_account_in_hq" event happened. Reviewing that email exchange though makes me think that that requirement might not affect the workflow of those people who are using hubspot. I think it would be worth checking with whoever the hubspot person is now if it's ok to create new contacts on any event, or if it new contacts should still only be created with users create an account.